server/routes.js

```diff
@@ -140,7 +140,8 @@ function getSettings(request, callback) {
 function getTransaction(request, callback) {
   var account = request.params.account;
   var identifier = request.params.identifier;
-  api.getTransaction(account, identifier, callback);
+  var options = request.query;
```

Comment:
Instead of passing in the entire `query` object, can we make a new object for options with just the ledger?
`var options = { ledger: request.query.ledger };`